adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Microsoft.IdentityModel.Clients.ActiveDirectory.csproj

```diff
@@ -1,7 +1,14 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras/1.5.4">
-
-  <PropertyGroup>
+  <Target Name="MacOSBuild" BeforeTargets="Build">
+    <Message Text="MAC_OS_BUILD = $(MAC_OS_BUILD)" Importance="High" />
+  </Target>
+  <PropertyGroup Condition="'$(MAC_OS_BUILD)' == ''">
```

Comment:
You have an msbuild variable that you can use - see https://github.com/Microsoft/msbuild/issues/539

Reply:
Great. ill use this instead